README.md

```diff
@@ -18,16 +18,16 @@
 It supports optional parameters:
 
 - `mode` - enum (`bytes`, `rows`, `slices`), default `bytes`
-- `bytesPerSlice` (`int`) - for `mode = bytes`, maximum size of the one slice in bytes before compression, default `524 288 000` (`500 MiB`)
+- `bytesPerSlice` (`string`/`int`) - for `mode = bytes`, maximum size of the one slice in bytes before compression, default `500MB`
 - `rowsPerSlice` (`int`) - for `mode = rows`, maximum rows in the one slice, default `1 000 000`
 - `numberOfSlices` (`int`) - for `mode = slices`, fixed number of slices, default `60`
-- `minBytesPerSlice` (`int`) - for `mode = slices`, minimum size of the one slice in bytes before compression, default `4 194 304` (`4 MiB`)
+- `minBytesPerSlice` (`string`/`int`) - for `mode = slices`, minimum size of the one slice in bytes before compression, default `4MB`.
 - `gzip` (`bool`) - enable gzip compression, default `true`
```

Comment:
Do konfiguracie procesora bolo mozne zadat `bytes` iba ako `int`.
- `500MB` sa blbo zadava ako cislo, jednoducho sa spravi chyba.
- Nechcem aby sa to do CLI zadavalo ako cislo.
- Predo som sem pridal `datasize` library, a je mozne zadat `bytes` ako string.
- Je to BC a funguje aj `int`, vid testy.